selfservice/strategy/password/.schema/login.schema.json

```
@@ -22,5 +21,25 @@
     "method": {
       "type": "string"
     }
+  },
+  "if": {
+    "properties": {
+      "password_identifier": {
+        "minLength": 1
+      }
```

```suggestion
      "password_identifier": {}
```

No need for any further checks here, an empty schema is always valid (and therefore this whole thing matches if the key exists).

---

This is to improve the error reporting in the UI to "length must be > 0" instead of "field is missing" iirc